test.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require './utils'
+
+def fetch_albums
+  File.readlines('discography.txt').map(&:chomp)
+end
+
+def get_albums_by_decade(albums)
+  result = {}
+  albums.each do |album|
+    year = album[0..2].to_i * 10
+    next if album.nil?
```

Comment:
This check should be the first thing you do if the collection can have nil values. I would also suggest just using `compact` con the collection to remove nil values before starting the iteration